src/applications/mhv/secure-messaging/containers/FolderThreadListView.jsx

```diff
@@ -287,7 +287,11 @@ const FolderThreadListView = props => {
     <div className="vads-u-padding--0">
       <div className="main-content vads-u-display--flex vads-u-flex-direction--column">
         <AlertBackgroundBox closeable />
-        {folder?.folderId === undefined && <LoadingIndicator />}
+        {folder === null ? (
+          <></>
+        ) : (
+          folder?.folderId === undefined && <LoadingIndicator />
+        )}
```

Comment:
I think the previously written statement would not render anyhting if `folder === null` anyways, it may be not needed to introduce this ternary operator, unless I am missing anything here

Reply:
For this scenario when the 503 alert is present we are no longer displaying the loading spinner. The previous displayed the spinner even when folderId was no longer undefined and switched to null... I will include pics👍

Reply:
interesting. In that scenario `when folderId was no longer undefined and switched to null` I guess `folder` would not be `null` neither which may not satisfy the scenario too. But if you are confident with this, that is ok. Just wanted to make sure we are not introducing unnecessary condition statement